wiki/Ranking_Criteria/en.md

```diff
@@ -78,7 +78,10 @@ Changes to the rules and guidelines in this document are proposed and discussed
 - **[Guest difficulty](/wiki/Beatmap/Guest_difficulty) creators, storyboarders, skinners and hitsounders must be added to the tags of a beatmap.** This is to give credit where credit is due and help others identify the main contributors of any given beatmap. Usernames containing single characters separated by spaces must have the spaces replaced with underscores.
 - **[Primary metadata sources](/wiki/Beatmap/Primary_metadata_source) must be used as references for metadata.** Do not modify metadata from primary sources except to comply with formatting and standardisation rules below. If no sources are available, use what is most common and recognizable.
 - **The artists of a song must be traceable to existing people.** If there is no existing person recorded to be the artist, then `Unknown Artist` is to be used. A song's sole artist cannot be a fictional character or program unless it is the artist's alias.
-- **You must use the Source field if the song comes from or is directly tied to another media such as a video game, movie, series, web site, community etc.** If the song was featured or tied to a media after it was released, the source field should not be used. If a song has multiple potential sources, any option is valid. For remixes, arrangements, or covers based upon the original song, the source(s) of the original song would apply in the same way.
+- **You must use the Source field if the song comes from or is directly tied to another media such as a video game, movie, series, etc.** If the song was featured or tied to a media after it was released, the source field is optional. If a song has multiple potential sources, any options are valid. 
+  - For remixes, arrangements, or covers based upon the original song, the source(s) of the original song would apply in the same way.
+  - Songs specifically made for events such as contest entries or event theme songs may use the event name as a source.
```

Comment:
why not just add the word "event" to the first line, alongside "video game"?

Reply:
will do